gossip/itemsfetcher/fetcher_test.go

```diff
@@ -0,0 +1,56 @@
+package itemsfetcher_test
+
+import (
+	"github.com/Fantom-foundation/lachesis-base/gossip/itemsfetcher"
+	"testing"
+	"time"
+)
+
+func TestFetcher(t *testing.T) {
```

Comment:
When I undo the change in `fetcher.go:179` this test is still passing. Is it intended to detect the issue fixed in this line?